Carrot-Assistant/app.py

```diff
@@ -248,7 +248,7 @@ async def run_db(request: PipelineRequest) -> List[Dict[str,Any]]:
     Parameters
     ----------
     request: PipelineRequest
-        An API request containing a medication name
+        An API request containing a list of informal names for medications and the options of a pipeline
```

Comment:
Could this comment be generic to other informal names, rather than just medication, or would we need to perform a larger refactoring of the tool for that?

Reply:
At the moment the SQL just runs queries against drug concepts. Extending it to other domains makes the search space much bigger, so a query will be a lot slower (2 minutes/query). There might be other optimizations we could make, but restricting it to drugs was an easy win, bringing queries down to 2-4 seconds.